docs/how-to/document-search/use_reranker.md

```diff
@@ -0,0 +1,44 @@
+# How-To: Use Reranker
```

Comment:
higher level comment: maybe we should consider moving rerankers to ragbits core ? i could see some potential use cases of them in other tasks too like eg. data augmentation

Reply:
I think that this is a good topic for the grooming. This task was about creating docs to the components that are already there, they might be moved in the future but I wouldn't move functionalities between packages in doc creation task

Reply:
agree - just wanted to raise a topic here. comment is not directly related to PR itself. marking as resolved